cpp/ps0001/ps0001_Solution.cpp

```diff
@@ -1,31 +1,19 @@
-/* help your brother solve summation problems
-    155
-    940
-   +168
-  --------
-   1263
-*/
-
-
 #include<iostream>
 
 int main() {
+    // prompting user for 3 numbers to add 1/2
+    short number1;
+    short number2;
+    short number3;
 
-    int number1;
-    int number2;
-    int number3;
-
-    
-   
     std::cin >> number1;
-    std::cin  >> number2;
-    std::cin  >> number3;
-
+    std::cin >> number2;
+    std::cin >> number3;
 
+    // outputting the result of the summation process 3/3
     std::cout << "--------" <<std:: endl;
     std::cout << number1 + number2 + number3 << std::endl;
 
-
-
     return 0;
 }
+// great work 😎 8/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [ ] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code